pageserver/src/basebackup.rs

```diff
@@ -4,181 +4,257 @@
 //! TODO: this module has nothing to do with PostgreSQL pg_basebackup.
 //! It could use a better name.
 //!
+//! Stateless Postgres compute node is lauched by sending taball which contains on-relational data (multixacts, clog, filenodemaps, twophase files)
+//! and generate pg_control and dummy segment of WAL. This module is responsible for creation of such tarball from snapshot directry and
```

Comment:
Do we need the snapshot directory for anything anymore? With this PR, everything is created from the data stored in the repository, right?

Reply:
Yes, we do not need snapshot directory any more.

Reply:
> Yes, we do not need snapshot directory any more.

Ok, let's remove it altogether then. One less thing we call a "snapshot" :-D